packaging/installer/netdata-support-bundle.ps1

```
@@ -700,15 +942,50 @@ if (Test-Path $ConfDir) {
+                               -MaxEvents $ch.Max -ErrorAction SilentlyContinue
+        # Application is a shared channel: keep only Netdata's own records. The
+        # dedicated channels are already Netdata-only, so filtering them would
+        # drop records whose ProviderName resolves differently under ETW.
+        if ($ch.Log -eq 'Application') { $events = $events | Where-Object { $_.ProviderName -match 'Netdata' } }
+        if ($events) {
+            $events | Select-Object TimeCreated, ProviderName, LevelDisplayName, Message | Format-List
+        } else {
+            '(no events in this window, or the channel does not exist on this system)'
+        }
+    }
+} "Get-WinEvent: Netdata/* ETW channels + NetdataWEL + Application (last ${SinceHours}h)" -cap $LogCap
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Save-Cmd' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)